run-mlreco/run_spine.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+source ../util/reload_in_container.inc.sh
+source ../util/init.inc.sh
+
+source load_mlreco.inc.sh
+
+[ -z "$ARCUBE_SPINE_CONFIG" ] && export ARCUBE_SPINE_CONFIG="2x2_full_chain_240819.cfg"
+
+# Only export onwards if the vars are filled. The exports are a tip from Kazu and
+# required for NDLAr.
+[ -n "$ARCUBE_SPINE_NUM_THREADS" ] && export NUM_THREADS=$ARCUBE_SPINE_NUM_THREADS
+[ -n "$ARCUBE_SPINE_OPENBLAS_NUM_THREADS" ] && export OPENBLAS_NUM_THREADS=$ARCUBE_SPINE_OPENBLAS_NUM_THREADS
+
+outFile=${tmpOutDir}/${outName}.MLRECO_ANALYSIS.hdf5
```

Comment:
I'm wondering, do we want to keep `MLRECO_ANALYSIS` or go full `SPINE` here?

Reply:
Yeah it's a good suggestion. We'd want to update naming in `CAFMaker` accordingly too. I have `CAFMaker` changes coming down the pipe tomorrow so I can probably incorporate that.

Reply:
Implemented in 7510dce1c4747258325e4bd5ca650f32fdf29e2f